akka-docs/src/main/paradox/stream/operators/Source-or-Flow/completionTimeout.md

```diff
@@ -15,6 +15,17 @@ If the completion of the stream does not happen until the provided timeout, the
 If the completion of the stream does not happen until the provided timeout, the stream is failed
 with a `TimeoutException`.
 
+## Example
+
+This example reads the numbers from a source and do some calculation in the flow with a completion timeout of 10 milliseconds. It will throw `TimeoutException` if stream does not complete in given time.
```

Comment:
The second sentence sort of repeats what the descriptions says rather than explaining what is going on in the example.

What about something like:
"It will fail the stream, leading to failing the materialized @scala[`Future`]@Java[`CompletionStage`] from `reduce` if the stream has not completed mapping the numbers from the source when the timeout hits."